.github/workflows/run-unit-tests.yml

```diff
@@ -55,8 +55,8 @@ env:
 jobs:  
   api-breakage:
     if: ${{ inputs.with_api_check && github.event_name == 'pull_request' && !github.event.pull_request.draft }}
-    runs-on: ubuntu-latest
-    container: swift:jammy
+    runs-on: ubuntu-24.04
```

Comment:
Updating the container makes sense, but changing the `runs-on` will just lock us in to a version instead of moving ahead when GH updates the `-latest` tag.

Reply:
I mean we don't care about GitHub, we care more about Swift which needs manual work anyway ... but it doesn't make a big difference so it's fine by me either way